addons/payment/wizards/payment_link_wizard.py

```diff
@@ -42,21 +40,18 @@ def default_get(self, fields):
     description = fields.Char('Payment Ref')
     access_token = fields.Char(compute='_compute_values')
     company_id = fields.Many2one('res.company', compute='_compute_company')
-    payment_acquirer_id = fields.Many2one(
-        string="Payment acquirer",
-        comodel_name="payment.acquirer",
-        index=True,
-        domain=[('state', 'in', ['enabled', 'test'])],
+    payment_acquirer = fields.Selection(
```

Comment:
Why put a selection, if a company after lot of aquirer it is horrible to choose.

Many2one is better (you can search). If you want forgot edit or create in the view you can use :
```xml
<field name="payment_acquirer_id" options="{'no_create':True,'no_edit':True}"/>
```

Reply:
According to our backlog, the user shall have the choice to select "all" acquirers and a many2one doesn't allows to do that